source/draw/RasterMonochrome.ooc

```diff
@@ -33,6 +33,7 @@ RasterMonochrome: class extends RasterPacked {
 	create: func (size: IntSize2D) -> Image { This new(size) }
 	copy: func -> This {
 		result := This new(this)
+		result _buffer = ByteBuffer new(this _buffer size)
```

Comment:
On second note, it doesn't. The arguments are of type `RasterImage`, and they don't have a buffer (the subclass `RasterPacked` does). It would work if we changed the argument to be of type `RasterPacked`. Or, we could add an `init~fromRasterPacked` in `RasterPacked` which would be called instead. Or, all subclasses to `RasterPacked` create the buffer in their respective `Copy` method (like `RasterMonochrome` does here).
I don't feel too comfortable messing around here, so I leave it for someone else to decide.